### ISSUE TITLE
Multiple keys = tracking bits

### ISSUE THREAD

**Issue by martinthomson** [alxdavids/draft-privacy-pass#5]
In the security considerations, you have:

> TODO: Can client's flag bad server practices?

Given that use of multiple keys can be used to segment users (with arbitrary granularity even), it seems that this system depends critically on having some way to limit the number of keys that a server can use.

**Comment by alxdavids** [comment]
Yes, this needs to be addressed. There have been some limited discussions about using an approach similar to key transparency, but we are still yet to specify anything concrete.
